src/Backend/Core/Installer/ModuleInstaller.php

```diff
@@ -770,8 +774,18 @@ private function completeAndSavePageBlocks(array $blocks, int $defaultRevisionId
 
             $positions[$position][] = $block;
 
-            $revisionId = $block['revision_id'] ?? $defaultRevisionId;
+            $page = $block['page'] ?? $defaultPage;
+            if (isset($block['revision_id'])) {
+                /** @var PageRepository $pageRepository */
+                $pageRepository = BackendModel::get(PageRepository::class);
+                $page = $pageRepository->find($block['revision_id']);
+            }
+
             $extraId = $block['extra_id'] ?? null;
+            $type = $extraId === null ? PageBlockType::richText() : $this->getPageBlockTypeForModuleExtra($extraId);
+            if ($extraId !== null && $type->isRichText()) {
```

Comment:
Why does having an `extraId` and a `richText` type mean the extra doesn't exist?

Reply:
fixed in 5c00c2d